src/Illuminate/Support/Str.php

```diff
@@ -986,6 +986,19 @@ public static function createUuidsNormally()
         static::$uuidFactory = null;
     }
 
+    /**
+     * Transliterate a string into it's closest ASCII representation.
+     *
+     * @param  string  $string
+     * @param  string|null  $unkown
+     * @param  bool|null  $strict
+     * @return string
+     */
+    public static function transliterate($string, $unkown = '?', $strict = false)
```

Comment:
Should the `$unkown` argument be `$unknown`? https://github.com/voku/portable-ascii/blob/f1c1181c85e099b66713c7a36a6d65814795d149/src/voku/helper/ASCII.php#L1178

If accepted, most `Str` methods also get added to the `Stringable` class.

Reply:
I debated changing this and the strict flag. As both on their own are nondescript. However, decided to keep their original names for consistency. 

I could change `$unkown` to `$unkownFallback` and `$strict` to `$useIntl` as it's a bit more descriptive.

Reply:
Sorry, I meant that there's a typo missing the second 'n' in unk_n_own.

Reply:
D'oh! Two secs